pom.xml

```diff
@@ -99,8 +99,8 @@
     <wstx.version>6.6.2</wstx.version>
     <shrinkwrap.version>1.2.6</shrinkwrap.version>
     <derby.version>10.14.2.0</derby.version>
-    <arquillian.version>1.8.0.Final</arquillian.version>
-    <arquillian-wildfly-container.version>5.1.0.Beta2</arquillian-wildfly-container.version>
+    <arquillian.version>1.8.1.Final</arquillian.version>
```

Comment:
We need to upgrade to `<arquillian.version>1.9.0.Final</arquillian.version>` , which has https://github.com/arquillian/arquillian-core/pull/583 included, and  https://issues.redhat.com/browse/ARQ-2231 fixed.

Reply:
Done. Thanks for review @gaol